### PR TITLE
Version/3.26

### PR DESCRIPTION
**Pull request by lbudai**

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: NEWS.md [hunk 1]
```diff
@@ -71,7 +392,4 @@ of syslog-ng, contribute.
 
 We would like to thank the following people for their contribution:
 
-Andras Mitzki, Antal Nemes, Attila Szakacs, Balazs Scheidler, Clément Besnier,
-Gabor Nagy, jadhavsumit98, Janos Szigetvari, Laszlo Budai, Laszlo Szemere,
-László Várady, MikeLim, Nikita Uvarov, Norbert Takacs, pabloli, Péter Kókai,
-Zoltan Pallagi.
+<Fill this by the internal news file creating tool>
```
Comment (MrAnno) [inline NEWS.md]: Contributors: 

Andras Mitzki, Antal Nemes, Attila Szakacs, Balazs Scheidler, Dylan Perry, Gabor Nagy, Laszlo Budai, Laszlo Szemere, László Várady, Norbert Takacs, Peter Czanik, Péter Kókai, Romain Tartière, Tomáš Mózes.

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: NEWS.md (hunk 1 above)
Comment (alltilla) [reply NEWS.md]: Thanks! :)

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**